crates/bevy_ecs/src/observer/mod.rs

```diff
@@ -18,15 +18,21 @@ use std::marker::PhantomData;
 /// [`Event`] data itself. If it was triggered for a specific [`Entity`], it includes that as well.
```

Comment:
Should update the docs for this to say it includes propagation information.

Reply:
Done!